docs/ref/models/instances.txt

```diff
@@ -621,7 +621,7 @@ from the DB will get updated. In effect there is an automatic
 ``update_fields`` in this case. If you assign or change any deferred field
 value, the field will be added to the updated fields.
 
-.. note::
+.. admonition:: ``pre_save()`` is only called for ``update_fields``
```

Comment:
Maybe simply "Field.pre_save() and update_fields"?

Reply:
Thanks, updated.